### PR TITLE
[DNM] Deep Fryer now properly reflects contained reagents

### PR DESCRIPTION
**Pull request by Iamgoofball**
deepfrying something in polyacid will now make it look purple, and deep frying in tricord makes it look bubbly pink

probably shouldn't merge this until I find whatever the proc is that lowercases any input text for the batter name and doublecheck that corn oil has the old reagent color

done with Github Editor #wow #whoa because byond a shit right now

### CODE REVIEW

**Comment by 9600bauds** [comment]
>corn oil battered deep fried hamburger
doesn't have the same ring to it

**Comment by ComicIronic** [comment]
Why did you open a PR if your 'WIP' is 4 lines that need no coder review

**Comment by Iamgoofball** [comment]
Is this denied or just closed for now?

**Comment by ComicIronic** [comment]
It was just closed, because the PR itself was pointless.

**Comment by Iamgoofball** [comment]
ah

@9600bauds yeah, i was thinking of either making corn oil not show the message or rephrase it to sound better

**Comment by clusterfack** [comment]
Until finished, maybe?
